build.gradle

```diff
@@ -66,7 +66,8 @@ dependencies {
     implementation group: 'commons-codec', name: 'commons-codec', version: '1.5'
     implementation group: 'cz.msebera.android' , name: 'httpclient' , version: '4.4.1.2'
     implementation group: 'com.intellij', name: 'annotations', version:'9.0.4'
-    implementation group: 'com.google.android.gms', name: 'play-services-location', version:'8.3.0'
+    implementation group: 'com.google.android.gms', name: 'play-services-location', version:'11.6.0'
+    implementation "com.google.android.libraries.places:places-compat:1.1.0"
```

Comment:
Is this something like a gem file?

Reply:
Something like that - those are basically parts of the SDK, containing some specialized APIs. At least those specific ones, you can also use some other libraries, but in general yeah, they work pretty much like gems.